### PR TITLE
Feat/add most popular pact page

### PR DESCRIPTION
**Pull request by realdreamer**

### CODE REVIEW

File: packages/apps/docs/src/pages/docs/pact/index.tsx [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+  return {
+    props: {
+      popularPages: mostPopularPages,
+      leftMenuTree: checkSubTreeForActive(getPathName(__filename)),
+      frontmatter: {
+        title: 'Pact',
+        menu: 'Pact',
+        subTitle: 'The safest, most user-friendly language for smart contracts',
+        label: 'Introduction',
+        order: 0,
```
Comment (sstraatemans) [inline packages/apps/docs/src/pages/docs/pact/index.tsx:142]: order should be `2`.
the order of the index page is the order in the parentTree.
in its own tree its order will always be 0

**Review by eileenmguo — APPROVED** [review]

File: packages/apps/docs/src/pages/docs/pact/index.tsx [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+            <Card>
+              <Stack direction="column" spacing="$sm">
+                <Heading as="h6">Getting Started is Simple</Heading>
+                <Text as="p">
+                  Learn Kadena&apos;s core concepts & tools for development in
+                  15 minutes.
+                </Text>
+                <Stack direction="row" spacing="$sm">
+                  <Button
+                    title="Get Started"
```
Comment (sstraatemans) [inline packages/apps/docs/src/pages/docs/pact/index.tsx:46]: something a little off in responsive:
image

**Comment by Randynamic** [comment]
@sstraatemans @realdreamer what to do with this PR?

**Comment by realdreamer** [comment]
It's no more relevant. we updated in different PR

**Pull request closed without merging**